src/helper/style-path.js

```diff
@@ -8,9 +8,9 @@ const MIXED = 'scratch-paint/style-path/mixed';
 /**
  * Called when setting fill color
  * @param {string} colorString New color, css format
- * @param {!function} onUpdateSvg A callback to call when the image visibly changes
+ * @return {boolean} Whether the color application actually changed anything.
```

Comment:
Nit: 'Anything' is kind of vague, maybe 'any colors' or 'visibly' (in both places)

Reply:
Fixed in 3dab6ef